pkg/cluster/mocks/updater.go

```diff
@@ -0,0 +1,18 @@
+package mocks
+
+import (
+	"github.com/improbable-eng/thanos/pkg/cluster"
+	"github.com/improbable-eng/thanos/pkg/store/storepb"
+)
+
+type MetaUpdater struct {
+	Meta cluster.PeerMetadata
+}
+
+func (u *MetaUpdater) SetLabels(labels []storepb.Label) {
+	u.Meta.Labels = labels
+}
+func (u *MetaUpdater) SetTimestamps(lowTimestamp int64, highTimestamp int64) {
+	u.Meta.LowTimestamp = lowTimestamp
+	u.Meta.HighTimestamp = highTimestamp
+}
```

Comment:
Not sure we need this mock with some of the comments above, but generally doesn't seem like a good idea to pull in a dep on `pkg/cluster` into every package that wants a simple mock for this interface.

Reply:
well, this is separate package `pkg/cluster/mocks`

Reply:
Yes, but it depends on `pkg/cluster` for the `Meta` field. So that dep transitively applies to all packages including `pkg/cluster/mocks`.